entitlement/src/main/java/org/killbill/billing/entitlement/dao/BlockingStateDao.java

```diff
@@ -25,7 +25,8 @@
 import org.joda.time.DateTime;
 import org.killbill.billing.callcontext.InternalCallContext;
 import org.killbill.billing.callcontext.InternalTenantContext;
-import org.killbill.billing.catalog.api.Catalog;
+import org.killbill.billing.catalog.api.StaticCatalog;
```

Comment:
Here (and in other `entitlement` files), you don't need the `StaticCatalog` import anymore.

Reply:
ack.